CLI tool is not working

I get this error when running the CLI tool
```
$ insomnia-import
Error: Cannot find module './importers'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (~/.nvm/versions/node/v7.7.2/lib/node_modules/insomnia-importers/src/cli.js:5:19)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
```

--- a/src/cli.js
+++ b/src/cli.js
@@ -2,7 +2,7 @@
 
 const program = require('commander');
 const path = require('path');
-const importers = require('./importers');
+const importers = require('./import');
 const fs = require('fs');
 const {version} = require('../package.json');
 

